Add Los Rios Community Colleges

kcameron85 opened the pull request:
Los Rios Community Colleges are a group of community colleges in the Sacramento, California area. 
Link to Parent Organization: https://losrios.edu/
Link to Computer Information Science Program: https://www.arc.losrios.edu/academics/programs-and-majors/computer-information-science
I have included in the upload a screenshot of my email account showing that apps.losrios.edu hosts the email account.

philipto commented:
@kcameron85 lib/domains/los_rios/Email: Your request has an incorrect file path format, please ensure the request follows     the repository rules set out at the bottom of [this page](https://github.com/JetBrains/swot) and change     it accordingly or start a new one.     If you are unable to make a pull request with the instructions provided, please send an email to support@jetbrains.com mentioning the URL of this request in the message.

kcameron85 commented:
Hello @philipto,
I updated the file path to comply with the repository rules. I'm not sure what happened with my original upload but I think I fixed it.

philipto commented:
@kcameron85 I am sorry, the domain losrios.edu is blacklisted because the school allows to register an email address there even if you are not a student of the school. If you are the student of this school, please apply with official document (OFFICIAL DOCUMENT tab at https://www.jetbrains.com/shop/eform/students).
